### DISCUSSION TITLE
Add repository dispatch events to all workflows

### DISCUSSION THREAD

**Discussion by belhyun** [upptime/upptime#124] [Ideas] [answered]
Can I add types: [uptime] to uptime.yml and repository_dispatch by default? In the case of me, the monitoring is not performed every 5 minutes, so the github action is called every 5 minutes through jenkins. In the case of me, you can monitor more closely.
![image](https://user-images.githubusercontent.com/895026/102420827-53a2d200-4046-11eb-935c-a18de24e0583.png)

**Comment by AnandChowdhary** [comment, marked as answer]
This is a great idea, I've add the `repository_dispatch` event to all workflows by default. The `types` are based on the name of the workflow.
